index.html

```diff
@@ -0,0 +1,357 @@
+          <p class="records">Mr. P Official</p>
+          <p class="likeAndView">
+            <span>12M views</span> <span>6 months ago</span>
+          </p>
+        </div>
+      </div>
+      <div class="SinglePreview">
+        <div class="img">
+          <img src="./resources/thumbnail/preview7.png" alt="" />
+        </div>
+        <div class="textContetn">
+          <p class="title">Mr. P - Paloma (Official Video)</p>
+          <p class="subtitle">ft. Singah</p>
+          <p class="records">Mr. P Official</p>
+          <p class="likeAndView">
+            <span>12M views</span> <span>6 months ago</span>
+          </p>
+        </div>
+      </div>
+      <div class="SinglePreview">
+        <div class="img">
+          <img src="./resources/thumbnail/preview8.png" alt="" />
+        </div>
+        <div class="textContetn">
+          <p class="title">Mr. P - Paloma (Official Video)</p>
+          <p class="subtitle">ft. Singah</p>
+          <p class="records">Mr. P Official</p>
+          <p class="likeAndView">
+            <span>12M views</span> <span>6 months ago</span>
+          </p>
+        </div>
+      </div>
+      <div class="SinglePreview">
+        <div class="img">
+          <img src="./resources/thumbnail/preview5.png" alt="" />
+        </div>
+        <div class="textContetn">
+          <p class="title">Mr. P - Paloma (Official Video)</p>
+          <p class="subtitle">ft. Singah</p>
+          <p class="records">Mr. P Official</p>
+          <p class="likeAndView">
+            <span>12M views</span> <span>6 months ago</span>
+          </p>
+        </div>
+      </div>
+      <div class="SinglePreview">
+        <div class="img">
+          <img src="./resources/thumbnail/preview6.png" alt="" />
+        </div>
+        <div class="textContetn">
+          <p class="title">Mr. P - Paloma (Official Video)</p>
+          <p class="subtitle">ft. Singah</p>
+          <p class="records">Mr. P Official</p>
+          <p class="likeAndView">
+            <span>12M views</span> <span>6 months ago</span>
+          </p>
+        </div>
+      </div>
+      <div class="SinglePreview">
+        <div class="img">
+          <img src="./resources/thumbnail/preview7.png" alt="" />
+        </div>
+        <div class="textContetn">
+          <p class="title">Mr. P - Paloma (Official Video)</p>
+          <p class="subtitle">ft. Singah</p>
+          <p class="records">Mr. P Official</p>
+          <p class="likeAndView">
+            <span>12M views</span> <span>6 months ago</span>
+          </p>
+        </div>
+      </div>
+    </div>
+  </body>
+</html>
+     <!-- <div class="textContetn">
+          <p class="title">Mr. P - Paloma (Official Video)</p>
+          <p class="subtitle">ft. Singah</p>
+          <p class="records">Mr. P Official</p>
+          <p class="likeAndView">
+            <span>12M views</span> <span>6 months ago</span>
+          </p>
+        </div> -->
```

Comment:
- kindly, you should remove commented code. please refer to my previous comment about commented codes.